### PR TITLE
Some fixes for compiling and running.

### PR DESCRIPTION
**Pull request by blattms**
Even with this fixes it does not run in parallel, yet.

### CODE REVIEW

File: ebos/eclproblem.hh [hunk 1]
```diff
@@ -621,7 +621,7 @@ public:
                 comm.broadcast(buffer.data(), position, 0);
             }
         } else {
-            int size;
+            size_t size;
```
Comment (akva2) [inline ebos/eclproblem.hh]: this is wrong, an int is sent.

File: ebos/eclproblem.hh [hunk 2]
```diff
@@ -2522,7 +2522,7 @@ private:
             comm.broadcast(&size, 1, 0);
             comm.broadcast(buffer.data(), pos, 0);
         } else {
-            int size;
+            std::size_t size;
```
Comment (akva2) [inline ebos/eclproblem.hh:2525]: this is correct, but i'd rather send position above, ie, send the int.

File: ebos/eclproblem.hh (hunk 1 above)
Comment (blattms) [reply ebos/eclproblem.hh]: Indeed. that fix was wrong.

File: ebos/eclproblem.hh (hunk 2 above)
Comment (blattms) [reply ebos/eclproblem.hh:2525]: Please do that.

**Review by blattms — COMMENTED** [review]
Shall I update the PR with the corrections?

**Comment by akva2** [comment]
i have included the fixes in my branch.

**Pull request closed without merging**